core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -97,13 +116,23 @@ public InProcessTelemetryChannel(String endpointAddress, boolean developerMode)
     public InProcessTelemetryChannel(Map<String, String> nameAndValues) {
         boolean developerMode = false;
         String endpointAddress = null;
+        int maxQueueItemCount = DEFAULT_NUMBER_OF_TELEMETRIES_PER_CONTAINER;
+        int sendIntervalInMillis = TRANSMIT_BUFFER_DEFAULT_TIMEOUT_IN_SECONDS;
```

Comment:
1. Please use the full word "Milliseconds"
2. This variable is in milliseconds while the constant is in seconds so I guess you need to multiply by 1000